.github/workflows/bump-elastic-stack-snapshot.yml

```diff
@@ -34,5 +34,7 @@ jobs:
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
           pipeline: ./.ci/bump-elastic-stack-snapshot.yml
+          notifySlackChannel: "#ingest-notifications"
```

Comment:
we also have #fleet-notification channel

Reply:
```suggestion
          notifySlackChannel: "#fleet-notifications"
```